README.md

```diff
@@ -99,13 +99,34 @@ their charters. The TSC cannot mandate existing working groups alter their
 charters. If such a situation arises, the TSC may decide to revoke the group's
 charter.
 
+## Project Application Acceptance Criteria
+
+All applications to the TSC must follow the application template. They are to be
+submitted as a PR into the Planetary Software TSC repository where they may be
+clarified and built-upon by participation from members of the Planetary Software
+community. Any updates/modifications to an application are the responsibility
+of the applying entity. They will be reviewed and voted on by TSC members
+at regularly scheduled meetings. Any PRs related to applications will remain
+open until either accepted or rejected by the TSC, and will be either merged or
+closed, accordingly. 
+
 ## Merging Pull Requests in the TSC Repo
-The submitter of a pull request (PR) is responsible for making changes to the text of the PR.  Any changes to the PR can be suggested by others in the PR thread, but changes to the PR should be made by the PR author.  In order to merge a PR, it must satisfy both of these two conditions:
+
+The submitter of a pull request (PR) is responsible for making changes to the
+text of the PR.  Any changes to the PR can be suggested by others in the PR
+thread, but changes to the PR should be made by the PR author.  In order to
+merge a PR, it must satisfy both of these two conditions:
 
   1. have been open for 24 hours
   2. have two approvals
 
-Exceptions to the above are minor typo fixes or cosmetic changes that don't alter the meaning of a document.  Those edits can be made via a PR which only requires a single approval (not two, and not open for 24 h) to be merged.
+Exceptions to the above are minor typo fixes or cosmetic changes that don't
+alter the meaning of a document.  Those edits can be made via a PR which only
+requires a single approval (not two, and not open for 24 h) to be merged.
+
+Another exception to the above would be PRs related to the application
+acceptance process of new software projects into the TSC. This is covered in the
+"Project Application Acceptance Criteria" section.
```

Comment:
I think this paragraph is clearer if you just say that applications follow a different process and then link to the section. See this SO question (https://stackoverflow.com/questions/6695439/how-to-link-to-a-named-anchor-in-multimarkdown/7015050#7015050) for how to create portable links to sub-headings.

Reply:
This PR now has conflicts with master.

Jesse, can you please determine whether this has been overcome by #66, and we can just close this PR, or refactor the intent of these changes on top of master?

Reply:
Yep this is obsolete.